runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/CoordinateClue.java

```diff
@@ -158,6 +164,8 @@
 		.put(new WorldPoint(3188, 3933, 0), "Wilderness. Resource Area.")
 		.put(new WorldPoint(2997, 3953, 0), "Wilderness. Inside Agility Training Area.")
 		.put(new WorldPoint(3380, 3963, 0), "Wilderness. North of Volcano.")
+		.put(new WorldPoint(3051, 3736, 0), "East of the Wilderness Obelisk in 28 Wilderness.")
+		.put(new WorldPoint(2316, 3814, 0), "West of Neitiznot, near thee bridge.")
```

Comment:
`the`?

Reply:
Fat-fingered my 'e' keey :upside_down_face: